tasks/collect-data/tests/test-collect-data-with-data.yaml

```diff
@@ -41,6 +40,8 @@ spec:
               spec:
                 application: foo
                 target: foo
+                data:
```

Comment:
Can we add something here to also verify the scenario fixed in the bugfix? i.e. nested stuff in `data`.

Reply:
The RPA has nested stuff. I put a `one` key in the RP so it can be overwritten by the `one` key in the RPA. Is this what you meant?